dtmi/meshsystems/device/ophelia-1.json

```diff
@@ -0,0 +1,68 @@
+{
+  "@id": "dtmi:MeshSystems:device:ophelia;1",
+  "@context": "dtmi:dtdl:context;2",
+  "extends": "dtmi:MeshSystems:core:device;1",
+  "displayName": {
+    "en": "Ophelia"
+  },
+	"description": "Cellular, Wi-Fi and BLE guardian module",
```

Comment:
Should we adjust formatting for consistency ?

Reply:
Fixed formatting